Makefile

```diff
@@ -94,7 +94,7 @@ npm:
 
 ## clean: Clear the builds and temporary files.
 clean:
-	rm -f shared/data/smarter_encryption.txt shared/data/bundled/smarter-encryption-rules.json integration-test/artifacts/attribution.json:
+	rm -f build/.smarter_encryption.txt integration-test/artifacts/attribution.json:
```

Comment:
Is the `.` here intentional? I'm not sure we need to make this file hidden, I see a use-case for having files in the `build` dir which are used as part of the build, but not included in the final output. Perhaps we could even make that more explicit with a `build/intermediates` folder that this file could be put in. WDYT?

Reply:
Yea, it was intentional. I think the name makes sense, since folks who need to edit the file will know where to look, and other folks won't have extra noise in their build dir. I would be OK with a directory for such temporary files, if we keep it hidden too.